static/app/components/core/link/link.spec.tsx

```diff
@@ -1,6 +1,21 @@
 import {render, screen} from 'sentry-test/reactTestingLibrary';
 
 import {ExternalLink, Link} from 'sentry/components/core/link';
+import {FrontendVersionProvider} from 'sentry/components/frontendVersionContext';
+
+// Mock React Router's Link component to spy on its props
+jest.mock('react-router-dom', () => ({
```

Comment:
lol no

Reply:
we have a few ways now of verifying location reload was called. see `testableWindowLocation`